File: 3146. Permutation Difference between Two Strings.go
func findPermutationDifference(s string, t string) int {
	sIndex := make([]int, 26)
	tIndex := make([]int, 26)
	for i := range len(s) {
		sIndex[s[i]-'a'] = i
		tIndex[t[i]-'a'] = i
	}
	answer := 0
	for i := range 26 {
		answer += abs(sIndex[i] - tIndex[i])
	}
	return answer
}

func abs(i int) int {
	if i < 0 {
		return -i
	}
	return i
}

File: 3146. Permutation Difference between Two Strings.py
class Solution:
    def findPermutationDifference(self, s: str, t: str) -> int:
        sIndex = [0]*26
        tIndex = [0]*26
        for i in range(len(s)):
            sIndex[ord(s[i])-ord('a')] = i
            tIndex[ord(t[i])-ord('a')] = i
        answer = 0
        for i in range(26):
            answer += abs(sIndex[i]-tIndex[i])
        return answer